plugins/module_utils/identity/keycloak/keycloak.py

```diff
@@ -37,7 +37,7 @@
 from ansible.module_utils.six.moves.urllib.error import HTTPError
 from ansible.module_utils._text import to_native
 
-URL_REALMS = "{url}/realms"
+URL_REALMS = "{url}/admin/realms"
 URL_REALM = "{url}/realms/{realm}"
```

Comment:
So according to @kris2kris this also needs to be changed:
```suggestion
URL_REALM = "{url}/admin/realms/{realm}"
```